Dockerfile

```diff
@@ -1,9 +1,9 @@
-ARG ATLANTIS_BASE=ghcr.io/runatlantis/atlantis-base
-ARG ATLANTIS_BASE_TAG_DATE=latest
-ARG ATLANTIS_BASE_TAG_TYPE=alpine
-
-# Stage 1: build artifact
+# syntax=docker/dockerfile:1
```

Comment:
Please run hadolint on this new Dockerfile and show output to confirm that that check is still passing

Reply:
setup hadolint